### PR TITLE
[RHACS] Added storage requirements for Scanner and Scanner DB

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Based on input from Engineering.
- Apples to 3.69

<hr>

NOTE:

- This PR is for RHACS docs.
- It doesn't require any special labels or milestones.

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@c-du 

1. Can you please check these updates?
2. Which versions of RHACS does this apply to?
   - [ ] 3.65
   - [ ] 3.66
   - [ ] 3.67
   - [ ] 3.68

File: modules/scanner-requirements.adoc [hunk 1]
```diff
@@ -6,19 +6,42 @@
...
 
 {product-title} includes an image vulnerability scanner called Scanner.
-This service scans images that are not already scanned by scanners integrated into image registries.
+This service scans images that are not already scanned by scanners integrated into image registries. Scanner also stores information about vulneribilities and scanned images in Scanner DB.
 
 [discrete]
-== Memory and storage requirements
+== Memory and storage requirements for Scanner
+
+Scanner can scale to multiple instances. Red Hat recommends running 2 scanner instances with the following memory and storage requirements:
```
Comment (c-du) [inline modules/scanner-requirements.adoc:14]: CPU resource also?

**Comment by c-du** [comment]
> @c-du
> 
> 1. Can you please check these updates?
> 2. Which versions of RHACS does this apply to?
>    
>    * [ ]  3.65
>    * [ ]  3.66
>    * [ ]  3.67
>    * [ ]  3.68
>    * [X]  3.69

File: modules/scanner-requirements.adoc [hunk 2]
```diff
@@ -6,19 +6,42 @@
 = Prerequisites for installing Scanner
 
 {product-title} includes an image vulnerability scanner called Scanner.
-This service scans images that are not already scanned by scanners integrated into image registries.
+This service scans images that are not already scanned by scanners integrated into image registries. Scanner also stores information about vulneribilities and scanned images in Scanner DB.
```
Comment (sagidlow) [inline modules/scanner-requirements.adoc:9]:
```suggestion
This service scans images that are not already scanned into image registries. The Scanner stores information about vulnerabilities and scanned images in Scanner DB.
```
- The change for vulnerabilities is required, it was a misspelling. 
- I changed the sentences a little bit for easier reading. Let me know if this changes the meaning of the sentences.

File: modules/scanner-requirements.adoc [hunk 3]
```diff
@@ -6,19 +6,42 @@
...
 
 [discrete]
-== Memory and storage requirements
+== Memory and storage requirements for Scanner
+
+Scanner can scale to multiple instances. Red Hat recommends running 2 scanner instances with the following CPU, memory, and storage requirements:
```
Comment (sagidlow) [inline modules/scanner-requirements.adoc:14]:
```suggestion
You can scale the Scanner to multiple instances. Red Hat recommends running 2 scanner instances with the following CPU, memory, and storage requirements:
```
- Should the second instance of "scanner" be capitalized?

**Review by sagidlow — COMMENTED** [review]
Just a couple of nit things. The changes in the sentences are recommendations for easier reading. Let me know if you want to discuss this.

**Comment by gaurav-nelson** [comment]
Thank you @sagidlow for reviewing.

1. I've updated the sentence to the following:
   ```
   This service scans images not already scanned by other scanners that you might
   have integrated with your image registries. Scanner stores information about 
   vulnerabilities and scanned images in Scanner DB.
   ```
2. We have always reffered to ACS components without `the`. I'm keeping the same for consistency by not using `the` before Scanner. I'll take this up with team and probably update it later.

**Pull request merged**